project/nsw-state-library-amplify/project.json

```diff
@@ -9,7 +9,8 @@
   "allowTranscriptDownload": true,
   "gaCode": "",
   "adminEmails": [
-    "glen.crawford@reinteractive.net"
```

Comment:
We've manually designated individual users as admins via either the admin panel or Rails console. Never got around to tidying up that particular file.